cloud/users/users.md

```diff
@@ -6,38 +6,28 @@ title: Users in Centreon Cloud
 In Centreon Cloud, users/contacts can:
 
 * Log in to the Centreon web interface.
-* Receive notifications (if they are given the **Notification** role).
+* Receive notifications.
 
-All users can access their own account to change their password. All users can see all resources monitored by the platform.
+All users can access their own account to change their password (by clicking on the profile icon in the top right corner of the screen). All users can see all resources monitored by the platform. All users receive [notifications](../alerts-notifications/notif-configuration.md).
 
 ## User roles
 
-There are 4 roles for users in Centreon Cloud, each with a specific set of rights.
+There are 3 roles for users in Centreon Cloud, each with a specific set of rights.
 
-* **Operators**. Typically, **Operators** are the people who will do the monitoring. They can:
+* **User**. Typically, **Users** are the people who will do the monitoring. They can:
   * use the **Resources Status** page : they can view the status of all resources, filter the view, acknowledge alerts, define/plan downtimes, force a check, add a comment.
   * use custom views, performance graphs and dashboards.
 
-* **Editors**. **Editors** can do the same things as **Operators**, but they also can:
+* **Editor**. **Editors** can do the same things as **Users**, but they also can:
   * create hosts and services, meta-services, templates, categories...
   * disacknowledge an acknowledgement, submit a result...
   * view the list of pollers, export the configuration.
 
-* **Administrators**. **Administrators** can do the same things as **Editors**, but they also can:
-  * create users and grant them rights
+* **Administrator**. **Administrators** can do the same things as **Editors**, but they also can:
   * create time periods
   * install plugin packs
   * view logs and information about the platform and the monitoring engine.
 
```

Comment:
En fait les admin pouvaient principalement gérer les utlisateurs. C'est dans le CCIAM maintenant.
La permission de "installer des pluggin packs" se raporte aux editor aussi en fait.
Les 2 autres (time period et logs) ont été laissé tombée. 
Du coup:
Editor: can do the same things as **Users**, but they also can:
  * create hosts and services, meta-services, templates, categories...
  * disacknowledge an acknowledgement, submit a result...
  * view the list of pollers, export the configuration.
  * install plugin packs
Administrator: can do the same things as **Editors**, but they also can:
- manage users